spec/system/visit_change_password_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe 'ChangePassword', type: :system do
+  include AuthSupport, CognitoSupport, NotifySupport
```

Comment:
do we need to include all three? NotifySupport should be enough I would guess

Reply:
I just removed AuthSupport & CognitoSupport and it seems okay. 👍

Reply:
could you squash the commits please? no point having two for the same change...

Reply:
Done